openmc/lattice.py

```diff
@@ -1315,10 +1476,123 @@ def from_xml_element(cls, elem, get_universe):
                 j += 1
         lat.universes = univs
         return lat
-
     def _repr_axial_slice(self, universes):
         """Return string representation for the given 2D group of universes.
 
+        The 'universes' argument should be a list of lists of universes where
+        each sub-list represents a single ring.  The first list should be the
+        outer ring.
+        """
+        if self._hextype:
+           return self._repr_axial_slice_ox(universes)
```

Comment:
Need four spaces for indentation
```suggestion
            return self._repr_axial_slice_ox(universes)
```